src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -69,7 +69,7 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
-        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName, IFileSystem fileSystem, IProcessServices processServices) {
+        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName, IFileSystem fileSystem, IProcessServices processServices, IVsStatusbar statusBar) {
```

Comment:
I guess it modeless but not cancellable. Does VS close, close solution, open another project works OK? You may want to consider modal version as in switching connections to reduce bug surface.